### PR TITLE
Change

### PR DESCRIPTION
**Pull request by DimaMzk**
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum

### CODE REVIEW

**Comment by DimaMzk** [comment]
This is a regular Comment

**Comment by DimaMzk** [comment]
# This Has Some Markdown 🤔

**Comment by DimaMzk** [comment]
image
This has an attachment

File: new.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+These are some cool changes, huh?
```
Comment (DimaMzk) [inline new.txt:1]: Ohhhhh

File: new.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+One day this will have keyword highlighting,
```
Comment (DimaMzk) [inline new.txt:3]:
```suggestion
One day this will have keyword highlighting, one day...
```

**Review by DimaMzk — COMMENTED** [review]
I just can't stop reviewing this PR, just so many things to critique!

**Review by DimaMzk — COMMENTED** [review]
Commenting through the API

**Review by DimaMzk — COMMENTED** [review]
Commenting through the API and closing

**Review by DHervochon — CHANGES_REQUESTED** [review]
This is terrible! Get good!

**Review by DHervochon — APPROVED** [review]
👍

**Comment by DHervochon** [comment]
Posting a comment that isn't a review.

**Comment by DimaMzk** [comment]
Haha wow another comment!

**Comment by DimaMzk** [comment]
This is a 

**Comment**

With multiple lines

```html
<p>Hello World</p>
```

**Comment by DHervochon** [comment]
test

**Pull request closed without merging**